### PR TITLE
[ON HOLD] PAH-150 | change route(s) in code and UI to drive(s) or unsyncDrives

### LINKED ISSUE #150: Change on frontend everywhere name from "Route" to "Drive"
This change is necessary because in the backend (Django Admin and populate database) there is "Drive" name and on the frontend, there is "Route" name. Now, this is inconsistent.

(split from #82)

### PR DESCRIPTION
**Pull request by juliaanholcer**
resolves #150

### CODE REVIEW

**Comment by juliaanholcer** [comment]
@monikakrawczak I can't assign you as a reviewer, but please review this PR. Please check if the changes didn't break something and if I didn't miss something.

File: frontend/src/store/index.js [hunk 1]
```diff
@@ -34,10 +34,11 @@ const store = new Vuex.Store({
   modules,
   mutations,
   plugins: [createPersistedState({
-    paths: [USER, ROUTES, CARS, LANGUAGE, ...Object.values(namespaces)],
+    paths: [USER, UNSYNC_DRIVES, CARS, LANGUAGE, ...Object.values(namespaces)],
   })],
   getters: {
     [IS_ONLINE]: state => state.isOnline,
+    [UNSYNC_DRIVES]: state => state.unsyncDrives,
```
Comment (adamgajzlerowicz) [inline frontend/src/store/index.js]: Can we please not use abbreviations?

**Comment by monikakrawczak** [comment]
hi @juliaanholcer! I've already checked your PR. Everything looks good for me! Name from 'route' into 'drive' is changes everywhere.

**Comment by adamgajzlerowicz** [comment]
merged under separate pr

**Pull request closed without merging**